src/applications/financial-status-report/components/CardDetailsView.jsx

```diff
@@ -26,10 +26,16 @@ const CardDetailsView = ({ formData, onEdit, index, title }) => {
   let age;
 
   keys.forEach(item => {
-    if (item.toLowerCase().includes('type')) {
+    if (
+      item.toLowerCase().includes('purpose') ||
```

Comment:
Would probably name the iterable something other than item just something more descriptive. You could probably also make a reusable func for the includes comparison you are doing. 

Just some readability feedback, every thing else looks great! 🚀